packages/example/src/modules/program/infra/variableSizeProgramInfo.ts

```diff
@@ -0,0 +1,228 @@
+    type: 'portrait',
+  },
+  {
+    id: '10',
+    title: 'Program 10',
+    image: Rabbit10,
+    description: 'Program 10 description',
+    type: 'portrait',
+  },
+  {
+    id: '11',
+    title: 'Program 11',
+    image: Rabbit11,
+    description: 'Program 11 description',
+    type: 'portrait',
+  },
+  {
+    id: '12',
+    title: 'Program 12',
+    image: Rabbit12,
+    description: 'Program 12 description',
+    type: 'square',
+  },
+  {
+    id: '13',
+    title: 'Program 13',
+    image: Rabbit13,
+    description: 'Program 13 description',
+    type: 'square',
+  },
+  {
+    id: '14',
+    title: 'Program 14',
+    image: Rabbit14,
+    description: 'Program 14 description',
+    type: 'square',
+  },
+  {
+    id: '15',
+    title: 'Program 15',
+    image: Rabbit15,
+    description: 'Program 15 description',
+    type: 'square',
+  },
+  {
+    id: '16',
+    title: 'Program 16',
+    image: Rabbit16,
+    description: 'Program 16 description',
+    type: 'square',
+  },
+  {
+    id: '17',
+    title: 'Program 17',
+    image: Rabbit17,
+    description: 'Program 17 description',
+    type: 'portrait',
+  },
+  {
+    id: '18',
+    title: 'Program 18',
+    image: Rabbit18,
+    description: 'Program 18 description',
+    type: 'portrait',
+  },
+  {
+    id: '19',
+    title: 'Program 19',
+    image: Rabbit19,
+    description: 'Program 19 description',
+    type: 'square',
+  },
+  {
+    id: '20',
+    title: 'Program 20',
+    image: Rabbit20,
+    description: 'Program 20 description',
+    type: 'square',
+  },
+  {
+    id: '21',
+    title: 'Program 21',
+    image: Rabbit21,
+    description: 'Program 21 description',
+    type: 'square',
+  },
+  {
+    id: '22',
+    title: 'Program 22',
+    image: Rabbit22,
+    description: 'Program 22 description',
+    type: 'square',
+  },
+  {
+    id: '23',
+    title: 'Program 23',
+    image: Rabbit23,
+    description: 'Program 23 description',
+    type: 'portrait',
+  },
+  {
+    id: '24',
+    title: 'Program 24',
+    image: Rabbit24,
+    description: 'Program 24 description',
+    type: 'square',
+  },
+  {
+    id: '25',
+    title: 'Program 25',
+    image: Rabbit25,
+    description: 'Program 25 description',
+    type: 'square',
+  },
+];
+
+const shuffleArray = <T>(array: Array<T>) => {
+  const arrayCopy = [...array];
+  for (let i = arrayCopy.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    const temp = arrayCopy[i];
+    arrayCopy[i] = arrayCopy[j];
+    arrayCopy[j] = temp;
+  }
+
+  return arrayCopy;
+};
+
+export const getVariableSizePrograms = (listSize?: number) => {
+  if (!listSize) return shuffleArray(programInfos);
+  const programInfosWithCustomSize: VariableSizeProgramInfo[] = [];
+  for (let i = 0; i < listSize; i++) {
+    programInfosWithCustomSize.push(
+      programInfos[Math.floor(Math.random() * (programInfos.length - 1))],
+    );
+  }
+  return programInfosWithCustomSize;
+};
```

Comment:
don't duplicate this fixture file, use this new one as a reference and adapt the existing places 😉

Reply:
Done :)